ov/enclosure.go

```diff
@@ -9,41 +9,74 @@ import (
+	DeviceBayCount                            int                   `json:"deviceBayCount,omitempty"`                            // "deviceBayCount": 16,
+	DeviceBays                                []DeviceBayMap        `json:"deviceBays,omitempty`                                 // "deviceBays": [],
+	DeviceBayWatts                            int                   `json:"deviceBayWatts,omitempty"`                            // "deviceBayWatts": 16,
+	ETAG                                      string                `json:"eTag,omitempty"`                                      // "eTag": "1441036118675/8",
+	EmBays                                    int                   `json:"emBays,omitempty"`                                    // "emBays": 16,
+	EnclosureGroupUri                         utils.Nstring         `json:"enclosureGroupUri,omitempty"`                         // "enclosureGroupUri": "/rest/enclosure-groups/293e8efe-c6b1-4783-bf88-2d35a8e49071",
+	EnclosureModel                            string                `json:"enclosureModel,omitempty"`                            // "enclosureModel": "Enclosure Group 1",
+	EnclosureType                             string                `json:"enclosureType,omitempty"`                             // "enclosureType": "BladeSystem c7000 Enclosure",
+	EnclosureTypeUri                          utils.Nstring         `json:"enclosureTypeUriomitempty"`                           // "enclosureTypeUri": "/rest/enclosure-groups/293e8efe-c6b1-4783-bf88-2d35a8e49071",
+	FanBayCount                               int                   `json:"fanBayCount,omitempty"`                               // "fanBayCount": 16,
+	FanBays                                   []FanBay              `json:"fanBays,omitempty`                                    // "fanBays": [],
+	FanAndManagementDevicesWatts              int                   `json:"fanAndManagementDevicesWatts,omitempty"`              // "fanAndManagementDevicesWatts": 16,
+	ForceInstallFirmware                      bool                  `json:"forceInstallFirmware,omitempty"`                      // "forceInstallFirmware": true
+	FrameLinkModuleDomain                     string                `json:"frameLinkModuleDomain,omitempty"`                     // "frameLinkModuleDomain": "",
+	FwBaselineName                            string                `json:"fwBaselineName,omitempty"`                            // "fwBaselineName": null,
+	FwBaselineUri                             utils.Nstring         `json:"fwBaselineUri,omitempty"`                             // "fwBaselineUri": null,
+	InterconnectBayCount                      int                   `json:"interconnectBayCount,omitempty"`                      // "interconnectBayCount": 8,
+	InterconnectBays                          []InterconnectBayMap  `json:"interconnectBays"`                                    // "interconnectBays": [],
```

Comment:
struct for InterconnectBayMap  is missing. If you are referring to already exiting struct in EG, then it doesnot have all attributes.

Reply:
Done